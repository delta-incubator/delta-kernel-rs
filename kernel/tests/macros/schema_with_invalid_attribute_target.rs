extern crate delta_kernel_derive;
use delta_kernel_derive::Schema;

mod actions;
mod schema;

#[derive(Schema)]
pub struct WithInvalidAttribute {
    #[drop_null_container_values]
    some_long_name: String,
}

fn main() {}
