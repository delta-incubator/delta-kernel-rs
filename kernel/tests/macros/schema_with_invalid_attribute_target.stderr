error: Can only use drop_null_container_values on HashMap fields, not String
  --> tests/macros/schema_with_invalid_attribute_target.rs:10:21
   |
10 |     some_long_name: String,
   |                     ^^^^^^
